CHANGELOG.md

```diff
@@ -67,11 +73,12 @@ Some support for Xcode Test Plans has been added. For now test plans are not gen
 
 ### Changed
 
-- Speed up source inclusion checking for big projects #1122 @PaulTaykalo 
+- Speed up source inclusion checking for big projects #1122 @PaulTaykalo
 
 ## 2.25.0
 
 ### Added
+
```

Comment:
There are a lot of auto-reformatting changes here, but I'm ok with them, as it's a stricter markdown